qemu/tests/cfg/image_locking_read_test.cfg

```diff
@@ -0,0 +1,37 @@
+- image_locking_read_test:
+    type = image_locking_read_test
+    virt_test_type = qemu
+    only qcow2
+    only linux
```

Comment:
Case cannot be parsed, can be executed after changing to **L**inux

Reply:
The following are the test results after changing the parameters:
 (1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.image_locking_read_test: PASS (548.37 s)(on P8 with -driver)
(3/3) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.image_locking_read_test: PASS (420.48 s)(on P9 with -blockdev)

Reply:
Yes, I limit this case to linux only.